src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/AsylumCaseDefinition.java

```diff
@@ -142,6 +142,12 @@
         "reviewTimeExtensionDecisionReason", new TypeReference<String>(){}),
     TIME_EXTENSIONS(
         "timeExtensions", new TypeReference<List<IdValue<TimeExtension>>>(){}),
+
+    PAYMENT_STATUS(
+        "paymentStatus", new TypeReference<String>(){}),
```

Comment:
just to confirm that you agree with Lee about PAYMENT_STATUS field name

Reply:
Yep, this may need to be "status" - I'll confirm with Lee